Source/Scene/Expression.js

```diff
@@ -56,8 +56,11 @@ define([
 
     });
 
-    Expression.prototype.evaluate = function(feature) {
-        return this._runtimeAst.evaluate(feature);
+    Expression.prototype.evaluate = function(feature, result) {
```

Comment:
I _think_ we would leave this function as is and then introduce a separate `evaluateColor` function that takes the result parameter.